recipes/gpjax/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "gpjax" %}
+{% set version = "0.12.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/gpjax-{{ version }}.tar.gz
+  sha256: f59a19f71f488a06182cbb4cf6bf51fcb8d97aef011f77fb6d1507c808269532
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.10,<=3.13
+    - hatchling
+    - pip
+  run:
+    - python >=3.10,<=3.13
```

```suggestion
{% set version = "0.12.0" %}
{% set python_min = "3.10" %}

package:
  name: {{ name|lower }}
  version: {{ version }}

source:
  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/gpjax-{{ version }}.tar.gz
  sha256: f59a19f71f488a06182cbb4cf6bf51fcb8d97aef011f77fb6d1507c808269532

build:
  noarch: python
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
  number: 0

requirements:
  host:
    - python >={{ python_min }}
    - hatchling
    - pip
  run:
    - python >={{ python_min }}
```

Using `{{ python_min }}` is easier to maintain. You have upper bounds on the Python version, which is coming from the project's `pyproject.toml` but as there aren't any compiled dependencies that can be specific to the Python release version this should get removed in general in favor of the known lower bounds only of Python 3.10.

---

Whilst I agree with the principle of these changes, they do appear have broken the Linux and OSX builds. Unfortunately, I cannot see the logs to understand why. Do you have any ideas?